#! /bin/sh
patch -p1 -l -f $* < $0
exit $?

From 9a6c805be0c2552fb38026d8136a8ea7657e187d Mon Sep 17 00:00:00 2001
From: Andre Heinecke <aheinecke@gnupg.org>
Date: Tue, 26 Mar 2019 19:46:57 +0100
Subject: [PATCH] Replace fancy attribute by comment

* src/mailitem-events.cpp: Change attribute fallthrough to magic
comment.

--
This fixed build with older gcc versions....
---
 src/mailitem-events.cpp | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/mailitem-events.cpp b/src/mailitem-events.cpp
index 5be58e7..c8e7125 100644
--- a/src/mailitem-events.cpp
+++ b/src/mailitem-events.cpp
@@ -761,11 +761,10 @@ EVENT_SINK_INVOKE(MailItemEvents)
           memdbg_dump ();
           TRETURN S_OK;
         }
-      /* Fallthrough */
       case ReplyAll:
       case Reply:
           is_reply = true;
-          __attribute__ ((fallthrough));
+          /* fall through */
       case Forward:
         {
           log_oom ("%s:%s: %s : %p",
--
2.20.1
